### PR TITLE
1801 objects settings add activate option to disabled menu

### LINKED ISSUE #1801: Objects Settings - Add "Activate" option to disabled menu
## Context

See #1762

### PR DESCRIPTION
**Pull request by bosiraphael**
- Created a dropdown menu with two menu item components : Activate and Erase
- The dropdown menu takes handleActivate and handleErase as props, but since this part is not yet connected to the backend, these functions will be implemented later
- Added stories : Default, WithOpen, WithActivate, WithErase

### CODE REVIEW

**Review by charlesBochet — APPROVED** [review]
LGTM with minor comment

File: front/src/modules/settings/data-model/objects/SettingsObjectDisabledMenuDropDown.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import { DropdownScope } from '@/ui/layout/dropdown/scopes/DropdownScope';
+import { MenuItem } from '@/ui/navigation/menu-item/components/MenuItem';
+
+type SettingsObjectDisabledMenuDropDownProps = {
+  scopeKey: string;
+  handleActivate: () => void;
+  handleErase: () => void;
+};
+
+const StyledDropdownMenuItemsContainer = styled.div`
```
[the remark this answers is not in the document]
Comment (bosiraphael) [reply front/src/modules/settings/data-model/objects/SettingsObjectDisabledMenuDropDown.tsx:16]: I will take a look!

[the remark this answers is not in the document]
Comment (bosiraphael) [reply front/src/modules/settings/data-model/objects/SettingsObjectDisabledMenuDropDown.tsx:16]: StyledDropdownMenu  in front/src/modules/ui/layout/dropdown/components/StyledDropdownMenu.tsx is the closest to what we need here doesn't define a padding and a gap. Do I can modify it to pass them as props ?

[the remark this answers is not in the document]
Comment (charlesBochet) [reply front/src/modules/settings/data-model/objects/SettingsObjectDisabledMenuDropDown.tsx:16]: you can also use DropdownMenuItemsContainer, it should work!

**Review by charlesBochet — APPROVED** [review]

**Pull request merged**